pypeit/wavecalib.py

```diff
@@ -677,16 +702,17 @@ def build_wv_calib(self, arccen, method, skip_QA=False,
             msgs.error('Unrecognized wavelength calibration method: {:}'.format(method))
 
         # Build the DataContainer
-        if self.par['redo_slit'] is not None:
+        if self.par['redo_slits'] is not None:
```

Comment:
Can you provide more comments on what is going on below. Presumably if redo_slits is being requested, you need to store some additional things, but I'm not following why these assignments are occurring only if redo_slits is not None.